subaccounts.go

```diff
@@ -44,30 +49,30 @@ func (c *Client) SubaccountCreate(s *Subaccount) (res *Response, err error) {
 func (c *Client) SubaccountCreateContext(ctx context.Context, s *Subaccount) (res *Response, err error) {
 	// enforce required parameters
 	if s == nil {
-		err = fmt.Errorf("Create called with nil Subaccount")
+		err = errors.New("Create called with nil Subaccount")
 	} else if s.Name == "" {
-		err = fmt.Errorf("Subaccount requires a non-empty Name")
+		err = errors.New("Subaccount requires a non-empty Name")
 	} else if s.KeyLabel == "" {
-		err = fmt.Errorf("Subaccount requires a non-empty Key Label")
+		err = errors.New("Subaccount requires a non-empty Key Label")
 	} else
 	// enforce max lengths
```

Comment:
Yeah, I did a double-take too. It seems to be the only way to make `go fmt` let you have a comment at that indentation level. It separates the different types of sanity checks, which we're removing in any case, so it's a [moo point](https://www.youtube.com/results?search_query=moo+point).